roles/provision_vm/tasks/deploy_from_template.yml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
I am not sure what's really the difference between those, but I think we can do but parametizing it via variables, so no need to have multiple tasks file here.

Reply:
I saw it is done in other validated content repos
I'm not sure sure about that

Reply:
It makes sense to have it in case module must be used bit differently, so we have multiple tasks file to suupert the use cases. But if module support the use cases well, there is no need. I am not sure if the module support it or not, let's check.

Reply:
The module supports it by using the template parameter